src/applications/login/components/SignInButtons.jsx

```diff
@@ -48,33 +48,37 @@ const LoginGovButtons = ({ isDisabled }) => (
       disabled={isDisabled}
       type="button"
       aria-label="Sign in with My HealtheVet"
-      className="usa-button usa-button-big mhv-button vads-u-margin-y--1p5"
+      className="usa-button mhv-button vads-u-margin-y--1p5"
       onClick={() => loginHandler('mhv')}
     >
       My HealtheVet
     </button>
     <div id="create-account">
       <h2 className="vads-u-margin-top--3">Or create an account</h2>
       <div className="vads-u-display--flex vads-u-flex-direction--column">
-        <button
-          type="button"
-          aria-label="Create an account with Login.gov"
+        <a
+          tabIndex="0"
+          role="link"
+          className="vads-c-action-link--blue logingov"
+          aria-label="Create an account with Login.gov. (Opens in a new window)"
           disabled={isDisabled}
           onClick={() => signup({ csp: 'logingov' })}
         >
           Create an account with Login.gov
-        </button>
-        <button
-          type="button"
-          aria-label="Create an account with ID.me"
+        </a>
+        <a
```

Comment:
Same feedback as previous link. This needs to either have a valid href or covert to a button element